plugins/commandParser.js

```diff
@@ -0,0 +1,223 @@
+    return [
+      { insertMessage: `Clean TTS: ${ _store_state[Chat.$states.cleanTts] }` }
+    ];
+  },
+
+  graph( stat, user ) { return [{ changeStatOnGraph: [stat, user] }] },
+
+  ignoreList() {
+    const _ignoreList = _store_state[Chat.$states.ignoreList];
+    const _ignoreChannelList = _store_state[Chat.$states.ignoreChannelList];
+    _ga.event( {
+      eventCategory: 'chat',
+      eventAction: 'ignorelist',
+    } ); // Analytics
+    return [
+      { insertMessage: `Ignored Users: ${ _ignoreList.join( ', ' ) }` },
+      { insertMessage: `Ignored Channels: ${ _ignoreChannelList.join( ', ' ) }` },
+    ]
+  },
+
+  async toggleBadge() {
+    const newstate = !_store_state[Chat.$states.showBadge];
+    await _store_commit( Chat.$mutations.setShowBadge, newstate );
+    return [{ insertMessage: `Badge ${ newstate ? 'on' : 'off' }` }]
+  },
+
+  skipTts() { speechSynthesis.cancel(); }, // Skip TTS
+
+  async bugReport() {
+    _sentry.withScope(
+      scope => {
+        scope.setExtra( 'global_chat', _store_state[ Chat.$states.global ] );
+        scope.setExtra( 'is_auth', _store.state[ VStore.namespace ][ VStore.$states.auth ] );
+        scope.setUser( {
+          username: this.username,
+        } );
+        this.$sentry.captureMessage( 'Bug Report' );
+      },
+    );
+    _sentry.showReportDialog( {
+      title: 'Something looks broken...',
+      labelName: 'Username',
+      labelSubmit: 'Submit Bug Report',
+    } );
+    return [];
+  }
+};
+
+const export_obj = {
+
+  commands: new Map([
+    ["dev", () => [{ chatServer: 'DEV' }, { insertMessage: 'Enabled developer mode.\nAttempting to connect to local dev server.' }]],
+    ["prod", () => [{ chatServer: 'PROD' }, { insertMessage: 'Disabled developer mode.\nAttempting to connect to production chat server.' }]],
+    ["production", () => [{ chatServer: 'PROD' }, { insertMessage: 'Disabled developer mode.\nAttempting to connect to production chat server.' }]],
+    ["local", () => { _store_commit( Chat.$mutations.setGlobal, false ); return []; } ],
+    ["global", () => { _store_commit( Chat.$mutations.setGlobal, true ); return []; } ],
+    ["ignore", functions.ignoreUser],
+    ["i", functions.ignoreUser],
+    ["unignore", functions.unignoreUser],
+    ["u", functions.unignoreUser],
+    ["ignorechannel", functions.ignoreChannel],
+    ["ic", functions.ignoreChannel],
+    ["unignorechannel", functions.unignoreChannel],
+    ["uic", functions.unignoreChannel],
+    ["uc", functions.unignoreChannel],
+    ["trolls", functions.hideTrolls],
+    ["susi", functions.hideTrolls],
+    ["cuckrockchris", functions.cleanTts],
+    ["crc", functions.cleanTts],
+    ["cleantts", functions.cleanTts],
+    ["graph", functions.graph],
+    ["ignorelist", functions.ignoreList],
+    ["badge", functions.toggleBadge],
+    ["skip", functions.skipTts],
+    ["s", functions.skipTts],
+    ["bugreport", functions.bugReport],
+  ]),
+
+  ...functions,
```

Comment:
They're here so you can access them with `this.$chatCommandParser.ignoreUser()` without forging a message to send to the parser.